### DISCUSSION TITLE
python venv select not work for dap debug

### DISCUSSION THREAD

**Discussion by meijies** [LazyVim/LazyVim#3064] [Q&A] [answered]
debug the python project, always use mason debugpy env, even chose the project env.
mason debugpy env: `~/.local/share/nvim/mason/packages/debugpy/venv`
project env: `{project_dir}/.venv.`

my trick operation is link project env to mason debugpy env and install debugpy, neovim package.

**Comment by se-jaeger** [comment, marked as answer]
Hi, I had the same problem and solve it as you can see here: https://github.com/se-jaeger/dotfiles/blob/main/dot_config/nvim/lua/plugins/python.lua

I basically disabled some dependencies, I don't use. The problam was that 'jay-babu/mason-nvim-dap.nvim' installs debugpy automatically and 'linux-cultist/venv-selector.nvim' then attaches to this venv.

Hope this helps.
